src/pybind/mgr/dashboard/services/nvmeof_client.py

```diff
@@ -9,6 +11,14 @@
 logger = logging.getLogger("nvmeof_client")
 
 try:
+    # if the protobuf version is newer than what we generated with
+    # proto file import will fail (because of differences between what's
+    # available in centos and ubuntu).
+    # this "hack" should be removed once we update both the
+    # distros; centos and ubuntu.
+    import os
+    os.environ["PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION"] = "python"
+
```

Comment:
This is not just due to a mismatch. Protobuf3 C++ is broken with subinterpreters, and this workaround is needed when running with that version (e.g. Fedora). The Protobuf4 C++ implementation should work properly.

See: https://github.com/protocolbuffers/protobuf/issues/10143